pkg/kubelet/cm/helpers_linux.go

```diff
@@ -40,6 +40,7 @@ import (
 const (
 	// Taken from lmctfy https://github.com/google/lmctfy/blob/master/lmctfy/controllers/cpu_controller.cc
```

Comment:
Maybe change this comment as the above link does not have anything about the value of MaxShares.

It also makes sense to refer to the kernel directly. Either of these links should work:
* https://github.com/torvalds/linux/blob/0bddd227f3dc55975e2b8dfa7fc6f959b062a2c7/kernel/sched/sched.h#L427-L428
* https://git.kernel.org/pub/scm/linux/kernel/git/tip/tip.git/tree/kernel/sched/sched.h?id=8cd9234c64c584432f6992fe944ca9e46ca8ea76#n269

Reply:
good idea, fixed!